modules/gapi/misc/python/package/gapi/__init__.py

```diff
@@ -11,6 +11,11 @@ def parameterized(func):
     return parameterized
 
 
+@register('cv2')
+def compile_args(*args):
```

Comment:
cv2 => cv2.gapi ?

(as there is no "G" prefix)

Reply:
but in `c++` it's a part of `cv` namespace, not `cv::gapi`

Reply:
It will be a bit weird to keep it in `cv.gapi` package in that case. Are u Ok to leave it as is ?

Reply:
@dmatveev Please confirm this usage.

Reply:
@alalek yes somehow we have `cv::compile_args` but not `cv::gapi::compile_args` in our C++ API.

It is definitely worth aligning in 5.0

Reply:
Then it makes sense to keep it in `cv.gapi` for python.

Reply:
Done